custom_components/powercalc/data/signify/1742930P7/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Signify, 1742930P7",
```

Comment:
This has to be the device which is used to measure the light. i.e. "Shelly Plug S"

Reply:
```suggestion
    "measure_device": "Shelly Plug S",
```